Objects/Framework/JPetGeantDecayTree.h

```diff
@@ -37,6 +36,10 @@ enum InteractionType
   kUnknownInteractionType
 };
 
+enum DecayChannel { 
+  Para2G, Direct2G, Ortho2G, Para3G, Direct3G, Ortho3G, Unknown
```

Comment:
I would suggest to add k to the enum names (e.g. `kPara2G`) since this is the convention that we are currently using in the Framework

Reply:
but maybe it is not relevant

Reply:
I will change the names of the enums